### PR TITLE
Sync configuration settings with `SHOW SETTINGS` output

### PR DESCRIPTION
**Pull request by stefano-ottolenghi**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-818.surge.sh

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -793,7 +827,7 @@ label:enterprise-edition[Enterprise Edition]
 |Description
 a|Advertised hostname/IP address and port for the RAFT server.
 |Valid values
-a|server.cluster.raft.advertised_address, a socket address in the format 'hostname:port', 'hostname' or ':port' which accessible address. If missing port or hostname it is acquired from server.default_advertised_address
+a|'hostname' or ':port' which accessible address. If missing port or hostname it is acquired from server.default_advertised_address
```
Comment (renetapopova) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:830]:
```suggestion
a|a socket address in the format 'hostname:port', 'hostname' or ':port' which accessible address. If missing port or hostname it is acquired from server.default_advertised_address
```

**Review by renetapopova — APPROVED** [review]
Looks good. I found one that wasn't trimmed correctly.

File: modules/ROOT/pages/configuration/configuration-settings.adoc (hunk 1 above)
Comment (stefano-ottolenghi) [reply modules/ROOT/pages/configuration/configuration-settings.adoc]: Good catch!

**Comment by stefano-ottolenghi** [comment]
Done already :)
https://github.com/neo4j/docs-operations/commit/a99bd5f5590523ac8b4ffda91c3b6960cc1d3f98

**Comment by renetapopova** [comment]
Ah, ok. Thanks! I'll link both PRs.

**Pull request merged**